Index: app/src/main/java/vn/loitp/app/activity/tutorial/rxjava2/AsyncTaskRxActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/vn/loitp/app/activity/tutorial/rxjava2/AsyncTaskRxActivity.java	(revision 391d4d2f88d11926e9f4f294f9973ee65578494d)
+++ app/src/main/java/vn/loitp/app/activity/tutorial/rxjava2/AsyncTaskRxActivity.java	(revision 44c5349ce59431d0494a1711fc43aecf03bf7501)
@@ -34,9 +34,8 @@
         super.onCreate(savedInstanceState);
         LLog.d(TAG,"1");
         LLog.d(TAG,"2");
-        LLog.d(TAG,"c");
-        LLog.d(TAG,"d");
-        LLog.d(TAG,"e");
+        LLog.d(TAG,"3");
+
         tv = (TextView) findViewById(R.id.tv);
         findViewById(R.id.bt_async_task).setOnClickListener(this);
         findViewById(R.id.bt_rx_1).setOnClickListener(this);
